apikeys.go

```diff
@@ -0,0 +1,133 @@
+package packngo
+
+import (
+	"fmt"
+)
+
+const (
+	apiKeyUserBasePath    = "/user/api-keys"
+	apiKeyProjectBasePath = "/projects/%s/api-keys"
+)
+
+// APIKeyService interface defines available device methods
+type APIKeyService interface {
+	UserList(*ListOptions) ([]APIKey, *Response, error)
+	ProjectList(string, *ListOptions) ([]APIKey, *Response, error)
+	UserGet(string, *GetOptions) (*APIKey, error)
+	ProjectGet(string, string, *GetOptions) (*APIKey, error)
```

Comment:
nit: I think these 2 should be grouped by User/Project

Reply:
I think you're right, the function names don't look good. I will make UserAPIKeyService and ProjectAPIKeyService, but keep it in one file.

Reply:
yeah I know.. But this way the code is slimmer and there's no duplication.